### PR TITLE
refactor collision matrix

### PR DESCRIPTION
**Pull request by JayceLai**
Re: cocos-creator/3d-tasks#4674

Changes:
 * refactor collision matrix

### CODE REVIEW

**Comment by pandamicro** [comment]
I think it should be merged in 3.0, or do you have a good reason ?

**Comment by JayceLai** [comment]
> I think it should be merged in 3.0, or do you have a good reason ?

ok

**Comment by JayceLai** [comment]
Waiting for rebase to be done

**Review by 2youyou2 — APPROVED** [review]

File: cocos/physics/framework/deprecated.ts [hunk 1]
```diff
@@ -36,6 +36,21 @@ replaceProperty(PhysicsSystem.prototype, 'PhysicsSystem.prototype', [
...
+        "name": "updateCollisionMatrix"
+    },
+    {
+        "name": "resetCollisionMatrix"
+    },
+    {
+        "name": "isCollisionGroup"
+    },
+    {
+        "name": "setCollisionGroup"
```
Comment (pandamicro) [inline cocos/physics/framework/deprecated.ts:53]: We can't remove these public APIs directly, please do a deprecation with the same functionality instead.

**Review by pandamicro — CHANGES_REQUESTED** [review]

**Review by pandamicro — APPROVED** [review]

**Pull request merged**